hyperspyui/mainwindow.py

```diff
@@ -31,9 +31,29 @@
 
 import numpy as np
 
-# Should go before any MPL imports:
-from hyperspyui.mainwindowhyperspy import MainWindowHyperspy, tr
+# Hyperspy uses traitsui, set proper backend
+from traits.etsconfig.api import ETSConfig
+try:
+    ETSConfig.toolkit = 'qt4'
+except ValueError:
+    if 'sphinx' not in sys.modules:
+        raise
+
+from qtpy import QtGui
+from qtpy.QtWidgets import QApplication, QSplashScreen
+from qtpy.QtGui import QColor, QPixmap
+from qtpy.QtCore import Qt
+splash_pix = QPixmap(os.path.join(
+    os.path.dirname(__file__), 'images', 'splash.png'))
+SPLASH = QSplashScreen(splash_pix, Qt.WindowStaysOnTopHint)
+SPLASH.show()
+SPLASH.showMessage("Initializing...", Qt.AlignBottom | Qt.AlignCenter |
+                   Qt.AlignAbsolute, QColor(Qt.white))
+QApplication.processEvents()
```

Comment:
This feels quite inappropriate to have in the global scope (executed on import). Why was this moved here?

Reply:
Maybe put it in a function that get's called in an appropriate spot?